### PR TITLE
MEN-3601: Install Mender deb package missing dependencies in fixture

### PR DESCRIPTION
**Pull request by lluiscampos**
-  [Python black] Enable style enforcement and update files

- MEN-3601: Install Mender deb package missing dependencies in fixture
This should fix integration tests at mender-binary-delta repo.

Follow-up from https://github.com/mendersoftware/mender-dist-packages/pull/30

### CODE REVIEW

File: conftest.py [hunk 1]
```diff
@@ -86,8 +86,10 @@ def setup_mender_configured(
             filename,
             key_filename=setup_test_container.key_filename,
         )
+        # Install deb package and missing dependencies
         setup_tester_ssh_connection.sudo(
-            "DEBIAN_FRONTEND=noninteractive dpkg -i %s" % filename
+            "DEBIAN_FRONTEND=noninteractive dpkg -i %s || sudo apt-get -f -y install"
```
Comment (oleorhagen) [inline conftest.py:91]: I might be missing something here, but `apt-get install` what?

Comment (lluiscampos) [reply conftest.py:91]: Not super intuitive, but it is a feature for `apt-get` to install upstream missing dependencies from the last `dpkg -i`.

See https://github.com/mendersoftware/mender-dist-packages/pull/30
See https://askubuntu.com/questions/40011/how-to-let-dpkg-i-install-dependencies-for-me

Comment (lluiscampos) [reply conftest.py:91]: @oleorhagen I pulled you into the review for the black part :)

Comment (oleorhagen) [reply conftest.py:91]: Then ten thumps up :+1: *10

**Pull request merged**